pkgs/development/tools/build-managers/bazel/common.nix

```diff
@@ -0,0 +1,563 @@
+{ version
+, sha256
+, src_deps
```

Comment:
This should probably also be camelCase like the rest of the attributes?

Reply:
Resolved